lib/consolidate.js

```diff
@@ -1179,14 +1179,14 @@ function reactRenderer(type){
         if (!cache(options)){
           // Parsing
           Code = (type === 'path') ? require(resolve(str)) : requireReactString(str);
```

Comment:
`require(resolve(str))` throws out error when using es6 `Class` module
>Invariant Violation: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object

[some similar issue](https://github.com/jedireza/hapi-react-views/issues/44)

change to `require(resolve(str)).default` could fix this error but may be bringing in new error for es5